src/main/java/ee/buerokratt/ruuter/controller/ConfigurationController.java

```diff
@@ -9,16 +9,18 @@
 import java.util.Map;
 
 import static org.springframework.http.ResponseEntity.ok;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
 @RestController
 @RequiredArgsConstructor
 public class ConfigurationController {
     private final ConfigurationService configurationService;
 
-    @RequestMapping(value = "/{configuration}", method = {RequestMethod.GET, RequestMethod.POST})
-    public ResponseEntity<RuuterResponse> queryConfiguration(@PathVariable("configuration") String code,
+    @RequestMapping(value = "/{configuration}", method = {GET, POST})
```

Comment:
Method types should be defined as setting variables.

https://github.com/orgs/buerokratt/projects/5/views/3

Reply:
Create a new task for it.

Reply:
Crated task for this spesific issue:
https://github.com/buerokratt/Ruuter/issues/41